### DISCUSSION TITLE
docs question: where to report vendor firmware issues?

### DISCUSSION THREAD

**Discussion by colemickens** [fwupd/fwupd#4348] [Q&A] [answered]
**Describe the question**

As I'm looking into devices, I'm noticing things like unsupported devices, or places where the firmware in fwupdmgr is older than firmware that I can find on the vendor's official support pages.

What should be done in this case? I'd like to sort of "leverage" fwupd and the community to centralize such identified improvements from vendors, but I also don't want to pollute your issue tracker with stuff that boils down to vendor process.

I checked the "End Users" page on the official site but didn't see much about this particular scenario.

Thanks again!

**Comment by colemickens** [comment]
Example: Dell XPS 13 9370:

For my NVME drive, for example, fwupd reports:
```
├─KXG50ZNV1T02 NVMe TOSHIBA 1024GB:
│     Device ID:          f954c7acdf5fab61aeaca1cd71d29ea5ade6992f
│     Summary:            NVM Express solid state drive
│     Current version:    AADA4106
...
```



* **laptop support page**: https://www.dell.com/support/home/en-us/product-support/product/xps-13-9370-laptop/drivers
  *  version listed as `2200.1070, A00`, super un-helpful
  *  filename is: Micron-2200S-PCIe-NVMe-Solid-State-Drive-Firmware_3JG3G_WIN64_2200.1070_A00.EXE``

* **generic dell/toshiba firmware page**: https://www.dell.com/support/home/en-us/drivers/driversdetails?driverid=7yxtm
  * lists `AADA4107, A00` 
  * filename is `Micron-2200S-PCIe-NVMe-Solid-State-Drive-Firmware_3JG3G_WIN64_2200.1070_A00.EXE`

Presumably `AADA4107` is newer than `AADA4106`.

So even for the actual laptop device, Dell knows there is a newer firmware, but it doesn't seem to be present in fwupd.

**Comment by superm1** [comment, marked as answer]
For Dell at least you should report them [here](https://github.com/fwupd/firmware-dell).

**Reply by colemickens** [reply]
Thank you @superm1 !
